tests/foreman/smoke/test_api_smoke.py

```diff
@@ -608,6 +611,19 @@
 }
 
 
+def _bz_1184170_affects_server():
```

Comment:
Mmhmm, I can see the benefit of that. I didn't bother doing anything more fancy because this method exists in response to a very specific bug, and the method will go away when that bug is resolved.